ntbs-service/wwwroot/source/Components/ValidateMultiple.ts

```diff
@@ -7,38 +7,56 @@ const qs = require("qs");
 const ValidateMultiple = Vue.extend({
     props: {
         model: String,

```

Comment:
@jasiekmiko Decided to extend this component after all - every other instance where we validate the full model has its own vue component and I didn't think we should add one for this page too. Nancy said that we'd be wanting to do date to from validation on several pages in future, so now we can use ValidateDate and ValidateMultiple in conjunction for all of these